client/src/components/new-request/search.vue

```diff
@@ -242,10 +242,15 @@ import { LocationT } from '@/models'
   components: { NameInput }
 })
 export default class NewSearch extends Vue {
-  /** Local Properties */
+  // Local Properties
   private corpNumValid: boolean = true
   private corpOnlineLink = 'https://www.corporateonline.gov.bc.ca/'
 
+  private mounted () {
+    // add classname to button text (for more detail in Sentry breadcrumbs)
+    this.$el.querySelector("#search-name-btn > span")?.classList.add("search-name-btn")
+  }
```

Comment:
Quick question about all the instances of these class assignments. Do the id's assigned in the templates not give it an accurate enough breadcrumb in Sentry?

Reply:
Sadly, no. Example:

![image](https://user-images.githubusercontent.com/10002889/103059618-778c7900-455a-11eb-90de-025a2e59e2da.png)

As  you can see, Sentry captures the span inside the button. That's why I added classnames to the span.

Reply:
Ah, yea that is a bummer. Looks like a fair bit of work here to add these in!